pkg/refactor/database/db_schema.sql

```diff
@@ -90,6 +90,33 @@ CREATE TABLE IF NOT EXISTS "secrets" (
     FOREIGN KEY ("namespace_id") REFERENCES "namespaces"("oid") ON DELETE CASCADE ON UPDATE CASCADE
 );
 
+DROP TABLE IF EXISTS "runtime_variables";
```

Comment:
I think this DROP surely needs to be removed? It's fine for unit tests but it shouldn't be in a release, right?